disnake/channel.py

```diff
@@ -224,6 +230,14 @@ def permissions_for(self, obj: Union[Member, Role], /) -> Permissions:
     def members(self) -> List[Member]:
         """List[:class:`Member`]: Returns all members that can see this channel."""
         return [m for m in self.guild.members if self.permissions_for(m).read_messages]
+    
+    @property
+    def can_send(self) -> bool:
+        return self.permissions_for(self.guild.me).send_messages
+    
+    @property
+    def can_read(self) -> bool:
+        return self.permissions_for(self.guild.me).read_messages
```

Comment:
Add a `with_history: bool = False` kwarg specifying whether read_history perm should also be checked

Reply:
There you go! Fixed it